.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.7.0.cjs
```

Comment:
Why a local instance? `npm install yarn -g` is not a way to go?

Reply:
⚠️ If there is something we developers must do to get it working with Yarn 3, please add it to the main README.

Reply:
I think it is already there. The main README file has a `Prerequisites` section with the link https://yarnpkg.com/ to the Yarn documentation. I do not think it is a good approach to copy steps from https://yarnpkg.com/getting-started/install to our docs because it can change in the future.

Information from Getting Started should be part of this PR description so you can easily migrate your development environment. But, after the merge, the prerequisites should be studied.

Reply:
This question remains:

> Why a local instance? ~~`npm install yarn -g` is not a way to go?~~

If I understand [the docs](https://nodejs.org/api/corepack.html) correctly, Corepack should pick up the right version online when it's not available locally. But I understand this can take some time in CI.

Reply:
@crishpeen Just for clarification, the `engines` field is meant to be for the SemVer range that specifies the Node.js you should use for the package to work with. Using this for defining the package manager can cause some trouble.